pyproject.toml

```diff
@@ -17,8 +17,9 @@ dependencies = [
   "rich==13.4.2",
   "requests==2.28.1",
   "requirements-parser==0.5.0",
-  "snowflake-connector-python==3.0.4",
-  "toml==0.10.2",
+#  "snowflake-connector-python==3.0.4",
```

Comment:
commented

Reply:
That's deliberate, we need to uncomment this asap